### PR TITLE
Update for FastBoot 1.0

### PR DESCRIPTION
**Pull request by simonihmig**
This makes it compatible with the latest ember-cli-fastboot 1.0.0-rc.1. Some context: https://github.com/ember-fastboot/ember-cli-fastboot/issues/387

Here I dropped support for the beta series of FastBoot (where separate browser/fastboot initializers were still available), so these changes only work with FastBoot >= 1.0.0 (-rc.x). As such this should get a minor bump to 0.1.0 when released!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
-  name: 'ember-data-fastboot'
+  name: 'ember-data-fastboot',
+
+  init: function() {
+    this._super && this._super.init.apply(this, arguments);
+
+    let checker = new VersionChecker(this);
+    let fastBootVersion = checker.for('ember-cli-fastboot');
+    if (!(fastBootVersion.gte('1.0.0') || fastBootVersion.gte('1.0.0-rc.1'))) {
+      let error  = new Error(`This version of ember-data-fastboot requires at ember-cli-fastboot above 1.0.0-rc.1, but you have ${fastBootVersion.version}. Revert to the latest 0.0.x version for support for beta versions of ember-cli-fastboot.`);
```
Comment (kratiahuja) [inline index.js:15]: You could just use silent-error instead of explicitly suppressing the stack trace.

**Review by kratiahuja — COMMENTED** [review]
LGTM

File: index.js (hunk 1 above)
Comment (simonihmig) [reply index.js:15]: You're right. Used that before. Copy pasted this from `ember-cli-version-checker`'s assert method...

**Comment by sivakumar-kailasam** [comment]
@MartinMalinda can we get this merged?

**Comment by MartinMalinda** [comment]
Released in 0.1.0

**Pull request merged**